dht.go

```diff
@@ -419,6 +431,21 @@ func makeRoutingTable(dht *IpfsDHT, cfg dhtcfg.Config, maxLastSuccessfulOutbound
 	return rt, err
 }
 
+// SetPrefixLength sets the prefix length for DHT provider lookups.
+func (dht *IpfsDHT) SetPrefixLength(prefixLength int) error {
```

Comment:
Is `SetPrefixLength` called somewhere in `go-libp2p-kad-dht`? Or is this function meant to be called by `kubo` or `go-libp2p` to configure the DHT settings?

Reply:
no, it's meant to be called externally. I needed this for testing, but thought it could be useful for users of the library also, since the only other way to set the prefix length is at DHT initialization.